sync.sh

```diff
@@ -24,20 +25,31 @@ Help()
 
 # Set variable defaults
 ALSO_COMPOSER_UPDATE=""
+ONLY_API=""
 
 # Get the options
-while getopts ":uh" option; do
+while getopts ":uha" option; do
    case $option in
       u)
          ALSO_COMPOSER_UPDATE="1";;
       h) # display Help
          Help
          exit;;
+      a) # only sync dist-persist api
+         ONLY_API="1";;
    esac
 done
 
 # Export vars for other scripts
 export ALSO_COMPOSER_UPDATE=${ALSO_COMPOSER_UPDATE}
+export ONLY_API=${ONLY_API}
+
+echo "Copy 'dist-persist' into the 'dist' folder"
+cp -rv ./dist-persist/* ./dist/
```

Comment:
I *think* reordering this is problematic. It probably needs to happen after the pacman stuff